qiskit/transpiler/passes/__init__.py

```diff
@@ -147,6 +147,7 @@
    HLSConfig
    SolovayKitaev
    SolovayKitaevSynthesis
+   AQCSynthesisPlugin
```

Comment:
Which other synthesis plugins are you referring to?

Reply:
Ah my bad -- I meant the `SolovayKitaevSynthesis`, I got confused because I was expecting two objects called `Plugin`. We should ensure these are named consistently in the future...